### PR TITLE
Change performance targets in roadmap

### PR DESCRIPTION
**Pull request by wojtek-t**
Change performance targets in roadmap to be more customer-visible.

cc @thockin @brendanburns @davidopp @fgrzadkowski

### CODE REVIEW

**Comment by wojtek-t** [comment]
To give a bit more context on that:
 I think that from the user perspective scheduling time is not important - what users pay attention to is e2e startup time. In other words, I think that even if we would schedule all pods in milliseconds, but starting them would take, say 1 minute, this won't produce a great user experience.

 Obviously, starting a pod involves loading an image, which we can't really control. but in my opinion, the goal should be something like that:
 "99 %ile of e2e pod startup time with all its images preloaded is less than X seconds; linear time to number of nodes and pods"

File: docs/roadmap.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ clustered database or key-value store. We will target such workloads for our
   - Status: in progress
 3. Scale to 30-50 pods (1-2 containers each) per node (#4188)
   - Status:
-4. Scheduling throughput: 99% of scheduling decisions made in less than 1s on 100 node, 3000 pod cluster; linear time to number of nodes and pods (#3954)
+4. Scheduling throughput: 99% of end-to-end pod startup time with prepulled images is less than 5s on 100 node, 3000 pod cluster; linear time to number of nodes and pods (#3952, #3954)
```
Comment (wojtek-t) [inline docs/roadmap.md]: BTW - I'm not 100% convinced that 5 seconds is a good target - please let me know if you have better suggestion for it.

**Comment by quinton-hoole** [comment]
i would suggest splitting into two separate performance targets (i.e. the original one, as well as the new proposed one).

File: docs/roadmap.md (hunk 1 above)
Comment (yujuhong) [inline docs/roadmap.md]: Why is this the scheduling throughput if we include kubelet pulling the image and starting the containers?

Comment (wojtek-t) [inline docs/roadmap.md]: It shouldn't be - updated. Thanks for pointing this out.

**Comment by wojtek-t** [comment]
Thanks Quinton - I agree this makes sense - updated.

**Comment by davidopp** [comment]
LGTM

**Comment by thockin** [comment]
LGTM - you guys can commit in your daytime hours :)

**Pull request merged**